src/main/resources/db/changelog/changeset/01-create-processing-table.yaml

```diff
@@ -0,0 +1,55 @@
+databaseChangeLog:
+  - changeSet:
+      id: 01-create-processing-table
+      author: freesky15
+      changes:
+        - createTable:
+            tableName: processingresults
+            columns:
+              - column:
+                  name: id
+                  type: bigint
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: arrivaltasktime
+                  type: timestamp
+                  constraints:
+                    nullable: false
+              - column:
+                  name: taskid
+                  type: bigint
+                  constraints:
+                    nullable: false
+              - column:
+                  name: solve
```

Comment:
solve means some raw user's solution?

Reply:
Yes, it is.

Reply:
Then noun sounds bettern than verb for the field name.

Reply:
Done!